src/foundry/common/utils/helpers.d.mts

```diff
@@ -414,6 +452,82 @@ export function randomID(length?: number): string;
  */
 export function timeSince(timeStamp: Date | string): string;
 
+/**
+ * Format a file size to an appropriate order of magnitude.
+ * @param size    - The size in bytes.
+ * @param options - Additional options. (default: `{}`)
+ * @returns
+ */
+export function formatFileSize(size: number, options: FormatFileSizeOptions): string;
+
+interface FormatFileSizeOptions {
+  /**
+   * The number of decimal places to round to.
+   * @defaultValue `2`
+   */
+  decimalPlaces?: number | undefined;
```

Comment:
Good job adding `| undefined` explicitly. We do this because of `exactOptionalPropertyTypes` in case you didn't know.

Reply:
I didn't, thanks :)